src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/UserService.kt

```diff
@@ -1,17 +1,20 @@
 package uk.gov.justice.digital.hmpps.manageusersapi.service
 
 import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.manageusersapi.adapter.auth.AuthApiService
 import uk.gov.justice.digital.hmpps.manageusersapi.adapter.external.UserSearchApiService
 import uk.gov.justice.digital.hmpps.manageusersapi.model.UserDetailsDto
-import java.util.Optional
 
 @Service
 class UserService(
+  private val authApiService: AuthApiService,
   private val externalUsersSearchApiService: UserSearchApiService
 ) {
-  fun findUserByUsername(username: String): Optional<UserDetailsDto> =
-    Optional.ofNullable(externalUsersSearchApiService.findUserByUsernameOrNull(username)?.toUserDetails())
-  // or nomis user
-  // or delius user
-  // or azure user
+  fun findUserByUsername(username: String): UserDetailsDto? =
+    externalUsersSearchApiService.findUserByUsernameOrNull(username)?.toUserDetails()
+      // or nomis user
+      // or delius user
+      ?: run {
+        authApiService.findAzureUserByUsername(username)
```

Comment:
As suggested in previous PR, now return null, rather than Optional (still to add nomis and delius calls)